### PR TITLE
implemented ai ranking of moves

### PR DESCRIPTION
**Pull request by tadhgj**
may consider using different ai but tbd

### CODE REVIEW

File: js/ai.js [hunk 1]
```diff
@@ -41,6 +41,45 @@ function getBestMove(grid, runs, debug) {
 		return {move: bestMove, score: bestScore};
 }
 
+function getBestMoveArray(grid, runs, debug) {
+	var moveArr = [0,0,0,0];
+	
+	for (var i = 0; i < 4; i++) {
```
Comment (maxmmyron) [inline js/ai.js]: can we use `let` or `const` :trollface:

**Review by maxmmyron — CHANGES_REQUESTED** [review]
Only one change necessary; if you want you could also remove the commented out lines but up to you.

Implementation is 🔥

File: js/ai.js (hunk 1 above)
Comment (tadhgj) [reply js/ai.js]: > can we use `let` or `const` :trollface:

:|

**Comment by tadhgj** [comment]
image

**Pull request merged**